### ISSUE TITLE
Run tests in Github CI

### ISSUE THREAD

**Issue by fgyanz** [SUSE/klp-build#49]
Hi team,

As you know, we've been greatly improving klp-build for some time now, making major refactors of the code and adding new functionalities. The testing has improved too, but it's not keeping up with so many changes. We need a way to automatically run tests in a reproducible and stable environment. Github CI should be the way to go, but it's not as straightforward as one would like. klp-build relies in many resources that are not yet publicly available, and as such we need to come up with some hacky workarounds. 

There's a on going draft PR trying to make klp-build work on github CI (See #39).

kind regards,
Fernando

**Comment by fgyanz** [comment]
Partially implemented on https://github.com/SUSE/klp-build/pull/89

**Comment by marcosps** [comment]
@fgyanz I believe that we can close it :)

**Comment by fgyanz** [comment]
yeah, although there's still the problem with setup and extract. Then again, we can just create another Issue explaining more in detail the situation :)
